### PR TITLE
Dashboard_periodical_executer is modified as follow :-

### PR DESCRIPTION
**Pull request by chopraapooja**
- Replaced old prototype style call (Class.create) for creating executer
    - Replaced Ajax call for JSON to jQuery.ajax call
    - Added pause condition for executor

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js [hunk 1]
```diff
@@ -43,7 +43,6 @@ BuildOutputObserver.prototype = {
 
         if (this.is_completed && this.is_output_empty) {
             this.update_page(json);
-            dashboard_periodical_executer.pause();
```
Comment (jyotisingh) [inline server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js]: Why did you get rid of this call? Won't it start making calls to update job status of a completed build?

Comment (chopraapooja) [inline server/webapp/WEB-INF/rails.new/app/assets/javascripts/build_output_observer.js]: Basically here, build_output_observer is pausing the executer, it is not the responsibility of observer to pause it's executor. Ideally executor should decide itself to pause on certain condition, I have moved this call [here] (https://github.com/chopraapooja/gocd/blob/job-status-json-polling-fix/server/webapp/WEB-INF/rails.new/app/assets/javascripts/dashboard_periodical_executer.js#L70-L72)

**Comment by jyotisingh** [comment]
Could you sync the the PR

**Comment by chopraapooja** [comment]
#1387 :+1:

**Pull request merged**